### ISSUE TITLE
Bootstrap 5.2 with dart-sass v1.56.0

### ISSUE THREAD

**Issue by Maaaleeek** [twbs/bootstrap#37465]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

After I update **dart-sass** from v1.55.0 to v1.56.0  [link](https://github.com/sass/dart-sass/releases/tag/1.56.0)  using Bootstrap v5.2.2, I get multiples warnings after `npm run build` like:

```
SassWarning: Deprecation $weight: Passing a number without unit % (100) is deprecated
To preserve current behavior: $weight * 1%

More information: https://sass-lang.com/d/function-units

nodes_modules/bootstrap/scss/_functions.scss 201:11      opaque()
nodes_modules/bootstrap/scss/_functions.scss 175:18      contrast-ratio()
nodes_modules/bootstrap/scss/_functions.scss 159:22      color-contrast()
nodes_modules/bootstrap/mixins/_forms.scss 22:19         form-validation-state()
nodes_modules/bootstrap/forms/_validation.scss 9:9       @import
nodes_modules/bootstrap/_forms.scss 9:9                  @import

```

### Reduced test cases

package.json

    "bootstrap": "^5.2.2",
    "sass": "^1.56.0",
    "sass-loader": "^13.1.0",
    "webpack": "^5.74.0",
    "webpack-cli": "^4.10.0",

### What operating system(s) are you seeing the problem on?

macOS

### What version of Bootstrap are you using?

v5.2.2

**Comment by julien-deramond** [comment]
Closing it. Duplicate of https://github.com/twbs/bootstrap/issues/37424 already fixed.

**Comment by dakur** [comment]
@julien-deramond Hi, I know you wrote in #37424 that this will be released in 5.3.0. But couldn't it be released as a patch earlier? Please see my question in https://github.com/twbs/bootstrap/discussions/37467. Thanks for consideration.

**Comment by julien-deramond** [comment]
Hey @dakur. We are discussing it in the core team, I'll provide an answer in https://github.com/twbs/bootstrap/discussions/37467 as soon as we have decided something.
